readme.md

```diff
@@ -707,6 +707,7 @@
 - [Executable Packing](https://github.com/dhondta/awesome-executable-packing#readme) - Packing and unpacking executable formats.
 - [Malware Persistence](https://github.com/Karneades/awesome-malware-persistence#readme) - Techniques that adversaries use to keep access to systems across restarts.
 - [EVM Security](https://github.com/kareniel/awesome-evm-security#readme) - Understanding the Ethereum Virtual Machine security ecosystem.
+- [Password Cracking](https://github.com/n0kovo/awesome-password-cracking#readme) - In cryptanalysis and computer security, "password cracking" refers to the process of recovering passwords from data that has been stored in or transmitted by a computer system in scrambled form.
```

Comment:
The description is too long

Reply:
2d0a885cf9c17c8385fba4158b9f4e90739aa5a2 Better?